.golangci.yml

```diff
@@ -3,19 +3,14 @@ run:
 linters:
   enable:
     - nakedret
-    - interfacer
-    - varcheck
-    - deadcode
-    - structcheck
```

Comment:
Could we also add [unused](https://github.com/golangci/golangci-lint/blob/master/pkg/golinters/unused.go), a suggested replacement for dead code and varcheck

Reply:
Already exists here: https://github.com/operator-framework/operator-sdk/pull/6351/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R15